pom.xml

```diff
@@ -927,10 +881,8 @@
           </systemPropertyVariables>
           <forkCount>1</forkCount>
           <reuseForks>false</reuseForks>
-          <rerunFailingTestsCount>1</rerunFailingTestsCount>
-          <includes>
-            <include>**/*Test.java</include>
-          </includes>
+          <rerunFailingTestsCount>${circleci.rerunFailingTestsCount}</rerunFailingTestsCount>
```

Comment:
Can we change the variable names to just be `ci` instead of `circleci` inside the poms - since they aren't specific to CircleCI.

Reply:
Done